lib/utils/format.ts

```diff
@@ -44,8 +44,11 @@ export const formatDate = (date: Date, dateInput = false): string => {
  * @param {Date} date date to be formatted
  * @returns {string} formatted date
  */
-export const formatDateYYYYMMDD = (date: Date): string => {
-  return new Date(date).toISOString().split('T')[0];
+export const formatDateYYYYMMDD = (d: Date): string => {
+  let date = new Date(d);
+  const offset = date.getTimezoneOffset();
+  date = new Date(date.getTime() - offset * 60 * 1000);
+  return date.toISOString().split('T')[0];
 };
```

Comment:
Noticed that this wasn't adjusting back to the right timezone. Converting to UTC would move the date forward or back by one at some points in the day
https://stackoverflow.com/questions/23593052/format-javascript-date-as-yyyy-mm-dd

Reply:
Great catch! This logic will likely get replaced when we do our upcoming date investigation, but for now it is a good solution